test_4_filter_numbers.py

```diff
@@ -1,6 +1,7 @@
 def filter_numbers(arr):
-  # Remove the `return None` code below and add your own code.
-  return None
+  newarr = [big for big in arr if big >= 10]
+  arr = [num for num in arr if num not in newarr]
```

Comment:
Haha the extra step in this code was one part me playing with list comprehension, and one part me just coding out my thought process and then not going back to revise it. I removed the newarr and replaced the code with the single line arr = [num for num in arr if num < 10]